src/pages/index.js

```diff
@@ -48,52 +78,116 @@ const cardListEl = new Section(
 cardListEl.renderItems();
```

Comment:
`cardListEl.renderItems();` this is not needed because it's an empty call

Reply:
without this call my initial cards do not load 
![image](https://github.com/user-attachments/assets/dc36ba06-cff6-42d7-b0b8-f026f8482412)
![image](https://github.com/user-attachments/assets/1277bbad-983e-404b-a4f5-01f91afc2ada)

Reply:
> without this call my initial cards do not load

You don't need the fake cards anymore. Only the `sever` cards should be rendered